src/components/ComputeModal.js

```diff
@@ -304,11 +307,15 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
       await Ajax().Disks.disk(googleProject, currentPersistentDiskDetails.name).delete()
     }
     if (shouldUpdateRuntime) {
-      await Ajax().Runtimes.runtime(googleProject, currentRuntimeDetails.runtimeName).update({ runtimeConfig })
+      await Ajax().Runtimes.runtime(googleProject, currentRuntimeDetails.runtimeName).update({
+        runtimeConfig,
+        autopauseThreshold: computeConfig.autopauseThreshold
+      })
     }
     if (shouldCreateRuntime) {
       await Ajax().Runtimes.runtime(googleProject, Utils.generateRuntimeName()).create({
         runtimeConfig,
+        autopauseThreshold: computeConfig.autopauseThreshold,
```

Comment:
Same comment as the update call. Is this supposed to be a field in the runtimeConfig, or a top-level field?

Reply:
It's supposed to be a top-level field, that is what Gabriela recommended when I paired with her

Reply:
I'd missed this. By "top-level", do you mean outside `compute/runtimeConfig` state? If so, I'd like to understand why `autopauseThreshold` is supposed to be top-level. Cc: @gpcarr

Reply:
<img width="420" alt="Screen Shot 2022-02-28 at 9 30 36 AM" src="https://user-images.githubusercontent.com/97535660/156000448-eefc570f-93ba-4fcc-af15-3f318538ef25.png">
autopauseThreshold is returned from the API call outside of the runtimeConfig, so we wanted to maintain that format on the Terra side as well

Reply:
I can easily remove it from runtimeConfig since it's not being used, I thought I had done that but must have forgotten. Otherwise I can refactor it into runtimeConfig and test that out, thoughts?

Reply:
I wonder if we may be conflating two distinct runtime configs. One is the state variable [computeConfig](https://github.com/DataBiosphere/terra-ui/blob/fa8d4a58fc9b6e0e25606ba1e63a6380c3c71a6d/src/components/ComputeModal.js#L213), which includes `autopauseThreshold` and there are also local `runtimeConfig`s like [this](https://github.com/DataBiosphere/terra-ui/blob/fa8d4a58fc9b6e0e25606ba1e63a6380c3c71a6d/src/components/ComputeModal.js#L266) that already include `autopauseThreshold`. Does that make sense?

Reply:
@kyuksel Leonardo is expecting the autopauseThreshold value to be outside of the runtimeConfig in the API calls so I believe it does still need to be separately added to the response

Reply:
Oh I think I see what you mean now. The duplicate is actually [line #266](https://github.com/DataBiosphere/terra-ui/blob/fa8d4a58fc9b6e0e25606ba1e63a6380c3c71a6d/src/components/ComputeModal.js#L266), which we should be able to remove?

Reply:
@kyuksel Yes, I just pushed that change. The backend leo changes have been approved and merged, so this should be testable now. Thanks!